websites/B/Broken Picturephone/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Luis Gabriel",
+    "id": "403648338286870529"
+  },
+  "url": "www.brokenpicturephone.com",
+  "description": {
+    "en": "A online game that you can draw, guess and send words to your friends guess."
+  },
+  "service": "Broken Picturephone",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/FYPmxU3.png",
```

Comment:
please get a better quality logo, or use the one you use as large image key.

Reply:
Both are the same, do you want that I update them to a best quality?

Reply:
![image](https://user-images.githubusercontent.com/34792838/93934196-14a10100-fd23-11ea-9f91-ce51a24a22bb.png)
I mean it looks really blurry so just something not blurry